### PR TITLE
README.md

### PR DESCRIPTION
**Pull request by mgarrixx**
This is the first draft of `README.md`

### CODE REVIEW

**Review by byanko55 — COMMENTED** [review]
You missed out demo link.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+├── Experience.yml
+├── Languages.yml
+├── Projects.yml
+├── Publications.yml
+├── Skills.yml
+```
+
+### Fill your infomation
+
+Open the *Yaml* file which you created right before.
```
Comment (byanko55) [inline README.md]: Seems like you'd better describe what one have to write in *Yaml*, before listing all the front matter.

**Comment by mgarrixx** [comment]
Check the demo page works

**Review by byanko55 — APPROVED** [review]
Looks nice. I'm just gonna close this PR.

**Pull request merged**